react/ButtonGroup/ButtonGroup.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ButtonGroup from './ButtonGroup';
+import Button from '../Button/Button';
+
+describe('ButtonGroup:', () => {
+  it('should render a button group', () => {
+    const wrapper = shallow(<ButtonGroup />);
```

Comment:
This test should throw a warning since children is a required prop, I'd rather test with 1it) with node  2it) with classname